### ISSUE TITLE
Thumbnail carousel swiping not working on mobile

### ISSUE THREAD

**Issue by drOleksandr** [leandrowd/react-responsive-carousel#382]
Hi,
Swiping thumbnails carousel on mobile (Chrome) not working. 
When I'm trying to do that I'm receiving following issue in the console:
![image](https://user-images.githubusercontent.com/33291077/69338692-66dd4580-0c6c-11ea-9846-6d8931bc88fa.png)

**Comment by erictakahashi** [comment]
This seems to be failing in the `Thumbs` component at this point:
```
        // if 3d isn't available we will use left to move
        if (this.itemsListRef) {
            ['WebkitTransform', 'MozTransform', 'MsTransform', 'OTransform', 'transform', 'msTransform'].forEach(
                (prop) => {
                    this.itemsListRef.style[prop] = CSSTranslate(position, this.props.axis);
                }
            );
        }
```

It looks like it's missing the swipe reference, so this appears to fix the problem:
```
this.itemsListRef.swiper.style[prop] = CSSTranslate(position, this.props.axis);
```

**Comment by tflearydev** [comment]
On the 'thumbs' class, try: 

.thumbs {
    
        overflow-x: auto !important;
        white-space: nowrap !important;
  
}

I implemented this, and it seemed to solve the problem for me. 

Personally, I created a jsx file for my carousel component (Carousel.jsx) and a standalone scss for it as well (Carousel.scss). The code above is in my scss file and imported into my Carousel.jsx, of course (import  './Carousel.scss'). Hope this helps
